src/main/main.ts

```
@@ -167,21 +168,25 @@
-    type: UpdateSourceType.StaticStorage,
-    baseUrl: `https://s3.aws.mulmocast.com/releases/${versionData}/${process.platform}/${process.arch}`,
-  },
-  logger: log,
-  notifyUser: true,
-  onNotifyUser: (info) => {
-    const lang = settingsManager.loadAppLanguage();
-    const notifyProps = config.messages[lang as keyof typeof config.messages].updater;
-    return makeUserNotifier(notifyProps)(info);
-  },
-});
+  const updateCallBack = (response: number) => {
```

ESLint

[error] 'response' is defined but never used. Allowed unused args must match /^__/u.

(@typescript-eslint/no-unused-vars)